packages/overlays/test/LocalOverlayController.test.js

```diff
@@ -445,6 +445,42 @@ describe('LocalOverlayController', () => {
       expect(controller._popper.options.modifiers.offset.offset).to.equal('0, 32px');
       expect(contentChild.style.transform).to.equal('translate3d(14px, -82px, 0px)');
     });
+
+    it('updates positioning correctly during shown state when config gets updated', async () => {
+      const controller = new LocalOverlayController({
+        contentTemplate: () =>
+          html`
+            <p>Content</p>
+          `,
+        invokerTemplate: () => html`
+          <button style="padding: 16px;" @click=${() => controller.show()}>
+            Invoker
+          </button>
+        `,
+        popperConfig: {
+          placement: 'top',
+        },
+      });
+      await fixture(html`
+        <div style="position: absolute; top: 300px; left: 100px;">
+          ${controller.invoker} ${controller.content}
+        </div>
+      `);
+
+      await controller.show();
+      const contentChild = controller.content.firstElementChild;
+      expect(contentChild.style.transform).to.equal('translate3d(14px, -58px, 0px)');
```

Comment:
Can we add a `msg` as a second argument to explain what is happening?

Reply:
Done ;)